ddtrace/internal/symbol_db/symbols.py

```diff
@@ -211,6 +212,7 @@ def _(cls, module: ModuleType, data: ScopeData):
             end_line=0,
             symbols=symbols,
             scopes=scopes,
+            language_specifics={"file_hash": sha1(module_origin.read_bytes()).hexdigest()},
```

Comment:
We have to produce a "git-compliant" hash, Git inserts a “blob <size>\0” prefix before computing the SHA-1.